src/content/content_instance.cc

```diff
@@ -145,9 +145,8 @@ void ContentInstance::PostAsyncSuccessReply(const picojson::value& msg,
 
 void ContentInstance::HandleSyncMessage(const char* message) {
   picojson::value v;
-  picojson::value::object o;
-
   std::string err;
+
```

Comment:
I don't think this is necessary, especially because those variables are all used in the following line anyway. As weird as it may look if you're not used to Chromium's coding style, it is geared towards minimizing [vertical whitespace](https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Vertical_Whitespace).